From 31f40175f702c8799bd69d5d9612d90e247501f3 Mon Sep 17 00:00:00 2001
From: "Miguel A. Vico" <mvicomoya@nvidia.com>
Date: Wed, 30 Mar 2016 15:23:49 +0200
Subject: [PATCH 03/11] gl-renderer: Rename <attribs> param of gl_renderer to
 <config_attribs>
X-NVConfidentiality: public

In preparation for follow-on changes to support frame presentation
through EGLDevice+EGLOutput, this change renames <attribs> parameter
of gl_renderer_display_create() and gl_renderer_output_window_create()
to <config_attribs>

Signed-off-by: Miguel A Vico Moya <mvicomoya@nvidia.com>
Reviewed-by: Andy Ritger <aritger@nvidia.com>
Reviewed-by: James Jones <jajones@nvidia.com>

[aplattner@nvidia.com: rebased on top of Weston 1.12.0]
---
 libweston/gl-renderer.c | 8 ++++----
 libweston/gl-renderer.h | 4 ++--
 2 files changed, 6 insertions(+), 6 deletions(-)

diff --git a/libweston/gl-renderer.c b/libweston/gl-renderer.c
index 0015268d210a..e11c22cf3f05 100644
--- a/libweston/gl-renderer.c
+++ b/libweston/gl-renderer.c
@@ -2547,7 +2547,7 @@ static int
 gl_renderer_output_window_create(struct weston_output *output,
 			  EGLNativeWindowType window_for_legacy,
 			  void *window_for_platform,
-			  const EGLint *attribs,
+			  const EGLint *config_attribs,
 			  const EGLint *visual_id,
 			  int n_ids)
 {
@@ -2557,7 +2557,7 @@ gl_renderer_output_window_create(struct weston_output *output,
 	EGLConfig egl_config;
 	int i;
 
-	if (egl_choose_config(gr, attribs, visual_id,
+	if (egl_choose_config(gr, config_attribs, visual_id,
 			      n_ids, &egl_config) == -1) {
 		weston_log("failed to choose EGL config for output\n");
 		return -1;
@@ -2891,7 +2891,7 @@ gl_renderer_create_pbuffer_surface(struct gl_renderer *gr) {
 
 static int
 gl_renderer_display_create(struct weston_compositor *ec, EGLenum platform,
-	void *native_window, const EGLint *attribs,
+	void *native_window, const EGLint *config_attribs,
 	const EGLint *visual_id, int n_ids)
 {
 	struct gl_renderer *gr;
@@ -2955,7 +2955,7 @@ gl_renderer_display_create(struct weston_compositor *ec, EGLenum platform,
 		goto fail_with_error;
 	}
 
-	if (egl_choose_config(gr, attribs, visual_id,
+	if (egl_choose_config(gr, config_attribs, visual_id,
 			      n_ids, &gr->egl_config) < 0) {
 		weston_log("failed to choose EGL config\n");
 		goto fail_terminate;
diff --git a/libweston/gl-renderer.h b/libweston/gl-renderer.h
index 5b3b68a8e4fa..9d6648680533 100644
--- a/libweston/gl-renderer.h
+++ b/libweston/gl-renderer.h
@@ -63,7 +63,7 @@ struct gl_renderer_interface {
 	int (*display_create)(struct weston_compositor *ec,
 		      EGLenum platform,
 		      void *native_window,
-		      const EGLint *attribs,
+		      const EGLint *config_attribs,
 		      const EGLint *visual_id,
 		      const int n_ids);
 
@@ -72,7 +72,7 @@ struct gl_renderer_interface {
 	int (*output_window_create)(struct weston_output *output,
 			     EGLNativeWindowType window_for_legacy,
 			     void *window_for_platform,
-			     const EGLint *attribs,
+			     const EGLint *config_attribs,
 			     const EGLint *visual_id,
 			     const int n_ids);
 
-- 
2.10.0

